src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -257,23 +260,33 @@ class RelationalDBInterfaceSqliteImp : public RelationalDBInterfaceSqlite
     }
 
     /**
-     * @brief checkoutLedger Checks if node ledger DB exists.
+     * @brief existsLedger Checks if node ledger DB exists.
      * @return True if node ledger DB exists.
      */
     bool
     existsLedger()
     {
-        return !!lgrdb_;
+        return static_cast<bool>(lgrdb_);
```

Comment:
Likely outside this PR's scope but since you are touching this function, it may be a good idea to make it const.